modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -14,3 +14,52 @@ source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/maven-s2i-overrides"
 # invoke the build
 maven_s2i_build
 
+#TODO: Clean this up to call the scripts instead
+
+shopt -s globstar
+echo "Setting environment variables"
+jarfile="/deployments/*.jar"
```

Comment:
this could glob to more than one jar, it's hard to know what the right one is. Possibly we can rely on `$JAVA_APP_JAR`. We will need to check that the variable is set by the s2i scripts if it wasn't set by the user.